RELEASE_PROCESS.md

```diff
@@ -72,13 +71,12 @@ The git SHA of the build you are testing can be found in
 
 ### Building and Releasing the Release
 
-- [ ] Select a version from the `current` channel that you wish to promote to `stable`.
-  Make sure that this version has gone through the upgrade testing. 
+- [ ] Select a version from the `current` channel that you wish to promote to `stable`. Make sure that this version has gone through the upgrade testing.
 - [ ] Use expeditor to promote the build:
 
         /expeditor promote chef/chef-server:master VERSION
 
-  Please do this in the `#chef-server-notify` room.  Once this is
+  Please do this in the `#chef-server-notify` room. Once this is
```

Comment:
erm - it is fine to do it in that room, we have been doing the promote in the #eng-infra-chef room, I think it might be better to do the promotions in the #chef-server room instead.

Reply:
Good point we should promote in the room where people work to enhance visibility. I updated the PR